src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -77,6 +77,9 @@ public function load($resource, $type = null)
         } catch (\Exception $e) {
             $this->loading = false;
             throw $e;
+        } catch (\Throwable $e) {
+            $this->loading = false;
+            throw $e;
```

Comment:
in php7 isn't possible implements ``\Throwable`` interface only extends `\Exception` class, so why not use only one catch with `\Throwable`?